pullRequestInsightsTask/AbstractAzureApi.ts

```diff
@@ -15,15 +15,27 @@ export abstract class AbstractAzureApi{
 
     public async abstract getMostRecentPipelinesOfCurrentType(project: string, currentPipeline: IPipeline, maxNumber: number, branchName: string): Promise<IPipeline[]>;
 
-    protected getConnection(): WebApi{
+    protected getConnection(): WebApi {
         return this.connection;
     }
 
-    public async postNewCommentThread(thread: azureGitInterfaces.GitPullRequestCommentThread, pullRequestId: number, repositoryId: string, projectName: string): Promise<void>{
-        (await this.getConnection().getGitApi()).createThread(thread, repositoryId, pullRequestId, projectName);
+    public async postNewCommentThread(thread: azureGitInterfaces.GitPullRequestCommentThread, pullRequestId: number, repositoryId: string, projectName: string): Promise<azureGitInterfaces.GitPullRequestCommentThread> {
+        return await (await this.getConnection().getGitApi()).createThread(thread, repositoryId, pullRequestId, projectName);
     }
 
-    public async getPullRequestData(repositoryId: string, pullRequestId: number, projectName: string): Promise<azureGitInterfaces.GitPullRequest>{
+    public async getCommentThreads(pullRequestId: number, repositoryId: string, projectName: string): Promise<azureGitInterfaces.GitPullRequestCommentThread[]> {
+        return (await this.getConnection().getGitApi()).getThreads(repositoryId, pullRequestId, projectName);
+    }
+
+    public async updateCommentThread(thread: azureGitInterfaces.GitPullRequestCommentThread, pullRequestId: number, repositoryId: string, projectName: string, threadId: number): Promise<void> {
```

Comment:
Should all of these return the object to remain consistent?

Reply:
changed return statements